packages/kit/src/exports/public.d.ts

```diff
@@ -430,6 +430,31 @@ export interface KitConfig {
 		 */
 		errorTemplate?: string;
 	};
+	/**
+	 * Image optimization configuration
+	 */
+	images?: {
+		/**
+		 * Path to a a file that contains a loader that will be used to generate the an image URL out of the given source and width.
+		 * It optionally also takes third parameter for options.
+		 *
+		 * ```js
+		 * export default function loader(src, width, opts) {
+		 *  return `https://example.com/${src}?w=${width}&q=${opts.quality || 75}`;
+		 * }
+		 * ```
+		 */
+		loader?: string;
+		/**
+		 * Which srcset sizes to generate
+		 * @default [640, 828, 1200, 1920, 3840]
+		 */
+		sizes?: number[];
+		/**
+		 * Which external domains to trust when optimizing images
```

Comment:
what's the usecase for this? it sort of seems like an oddly-specific feature. I imagine the user could create their own wrapper around `getImage` if this was something they needed

Reply:
It's needed if you want to optimize other images than ones that reside on your domain - else Vercel will not optimize them: https://vercel.com/docs/build-output-api/v3/configuration#images

Reply:
Ah, makes sense. But could we move these options to the adapter since they're somewhat Vercel-specific? It's a bit confusing to me that some options are here and some are in the adapter, so you need to check two places to see what the available options are. Maybe we should just put them all in the adapter?